src/collections/events/kubecon-eu-2021/index.mdx

```diff
@@ -5,6 +5,8 @@ thumbnail: ./kubecon-europe-2021.jpg
 eurl: https://kccnceu2021.sched.com/event/ijVr/cncf-sig-network-intro-and-deep-dive-lee-calcote-layer5-ken-owens-fiserv
 topic: event
```

Comment:
Can you also add `upcoming: true`  and `feature: true` here. And remove it from [here](https://github.com/layer5io/layer5/blob/master/src/collections/events/advanced-istio/index.mdx)

Reply:
sure, will do